src/Sylius/Bundle/WebBundle/Resources/views/Backend/Macros/misc.html.twig

```diff
@@ -41,6 +41,15 @@
     {% else %}
         {% set class = 'label-danger' %}
     {% endif %}
+{% elseif type == 'shipment' %}
+    {% set label = 'sylius.shipment.state.'~state %}
+    {% if state is same as('shipped') %}
```

Comment:
What is the benefit of using ``same as`` rather than simply ``==`` in this case?